automobile/vehiclewheel.md

```diff
@@ -1,20 +1,20 @@
-## AutomobileWheel
+## VehicleWheel
 
-The `AutomobileWheel` allows the user to easily create automobile wheels. It is
+The `VehicleWheel` allows the user to easily create vehicle wheels. It is
 designed to be generic and customizable to cover a wide range of wheel
-configurations. The base node of the `AutomobileWheel` PROTO is a `Slot` of type
-`automobile wheel` so that it can only be inserted as a wheel of an automobile
-PROTO featuring the corresponding `automobileWheel` slots.
+configurations. The base node of the `VehicleWheel` PROTO is a `Slot` of type
+`vehicle wheel` so that it can only be inserted as a wheel of a vehicle
+PROTO featuring the corresponding `VehicleWheel` slots.
```

Comment:
Before this was starting with a lower case letter.
Is this change made on purpose?

Reply:
No, it is a search and replace issue.